### DISCUSSION TITLE
Cannot connect to GPU backend

### DISCUSSION THREAD

**Discussion by mohdsh12** [lllyasviel/Fooocus#3378] [Q&A] [answered]
![Screenshot 2024-07-27 at 2 51 45 PM](https://github.com/user-attachments/assets/db737688-7674-49b4-aedb-f9c3930f9712)
I have a MacBook Pro 1 and this message was not showing up at first.

**Comment by mashb1t** [comment, marked as answer]
> You cannot currently connect to a GPU due to usage limits in Colab.

Click learn more to learn more.
